components/google_sheets/sources/new-row-added/new-row-added.js

```diff
@@ -63,90 +63,90 @@ module.exports = {
         );
       }
     },
-  },
-  async run(event) {
-    let subscription = this.db.get("subscription");
-    let channelID = this.db.get("channelID");
-    let pageToken = this.db.get("pageToken");
+    async processEvent(event) {
```

Comment:
Much of the processEvent method looks the same in both components. Couldn't the matching code be put in common.js?

Reply:
Done. Wanted to keep the scope to the bug only, but it's a good opportunity to do it while we're in that mental context given all the refactoring we've been doing 🙂